python/requirements/ml/rllib-test-requirements.txt

```diff
@@ -3,7 +3,6 @@
 # Environment adapters.
 # ---------------------
 # Atari
-gymnasium==0.28.1
```

Comment:
Since gymnasium is already part of the main Ray `requirements.txt` file, we won't need this here anymore.